app/views/devise/sessions/new.html.erb

```diff
@@ -1,6 +1,10 @@
 <h2>Log in</h2>
 
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <%= render 'devise/shared/error_messages', resource: resource %>
+  <% if flash[:alert] %>
+    <%= flash[:alert] %>
+  <% end %>
```

Comment:
did we not create a partial for flash? let's use that here, if not let's create a partial for flash